CMakeLists.txt

```diff
@@ -528,6 +529,16 @@ IF (PerlModular)
 	ENDIF()
 ENDIF()
 
+# scala modular
+IF (ScalaModular)
+	FIND_PACKAGE(Scala REQUIRED)
+	IF(NOT JavaModular)
+		IF(EXISTS ${CMAKE_SOURCE_DIR}/src/interfaces/java_modular)
+			add_subdirectory(${CMAKE_SOURCE_DIR}/src/interfaces/java_modular)
```

Comment:
please put an info message telling the user that java is enabled because scala needs that